doc/taoMediaManagerApi.yml

```
@@ -65,13 +88,38 @@ components:
             body:
               type: string
               example: '<?xml version="1.0" encoding="UTF-8"?><div xmlns="http://www.imsglobal.org/xsd/imsqti_v2p2"></div>'
+    SharedStimulusUpdate:
```

```suggestion
    SharedStimulusPatch:
```

Sorry, I forgot to mention this too.